docker-compose.yml

```diff
@@ -1,6 +1,11 @@
 version: '2'
 
 services:
+  tcpdump:
+    image: kaazing/tcpdump
+    network_mode: "host"
```

Comment:
why do we need     network_mode: "host"?

Reply:
> The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, or Docker EE for Windows Server.

https://docs.docker.com/network/host/

Reply:
I tried without it and it doesn't work. It doesn't show the network traffic. Maybe we can use a different type? I'll check. At the worst case, we submit a Linux VM so it can support it.